nistats/hemodynamic_models.py

```diff
@@ -513,12 +517,13 @@ def compute_regressor(exp_condition, hrf_model, frame_times, con_id='cond',
     # 4. temporally resample the regressors
     if hrf_model == 'fir' and oversampling > 1:
         computed_regressors = _resample_regressor(
-            conv_reg[:, oversampling - 1:], hr_frame_times[: 1 - oversampling],
-            frame_times)
+                conv_reg[:, oversampling - 1:],
```

Comment:
8-space indent ?

Reply:
So long as it's a multiple of 4, this is pretty common.

Reply:
Thx for the pointer, but _sorry to insist on such sow-level things_ I don't see the rationale here, since there is no possible confusion with nested indents.
Unfortunately, this pattern is present in lost of places in this PR. 
I'm usually not very picky on style considerations, but in view of the integration into Nielarn, that enforces this patterns, I really consider we should stick to the 4-space indents whenever this is not problematic.